dvc/command/config.py

```diff
@@ -14,6 +14,22 @@ def __init__(self, args):
         self.config = Config(validate=False)
 
     def run(self):
+        if self.args.list:
+            if any((self.args.name, self.args.value, self.args.unset)):
+                logger.error(
+                    "-l/--list can't be used together with any of these "
+                    "options: -u/--unset, name, value"
```

Comment:
Just consistency with the existing similar messages
```suggestion
                    "`-l/--list` can't be used together with any of these "
                    "options: `-u/--unset`, `name`, `value`"
```